spotlight-client/src/UiLibrary/Modal.tsx

```diff
@@ -0,0 +1,58 @@
+// Recidiviz - a data platform for criminal justice reform
+// Copyright (C) 2021 Recidiviz, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+// =============================================================================
+
+import { Modal as ModalBase } from "@recidiviz/case-triage-components";
+import { rem, rgba } from "polished";
+import styled from "styled-components/macro";
+import { colors } from ".";
+import animation from "./animation";
+import zIndex from "./zIndex";
+
+export const Modal = styled(ModalBase).attrs({
+  closeTimeoutMS: animation.defaultDuration,
+})`
+  /*
+    the double ampersands are a trick to overcome
+    specificity in the imported component styles
+  */
+  && .ReactModal__Overlay {
+    background: ${rgba(colors.modalOverlay, 0)};
+    transition: background-color ${animation.defaultDuration}ms,
+      backdrop-filter ${animation.defaultDuration}ms;
+    z-index: ${zIndex.modal};
+
+    &.ReactModal__Overlay--after-open {
+      /* not all browsers support backdrop-filter but it's a nice progressive enhancement */
+      backdrop-filter: blur(${rem(4)});
```

Comment:
it also would have been possible to apply a blur filter to the content itself, which is more broadly supported. In practice I found that the performance of this approach was quite bad, especially in combination with a CSS transmission (the animation was very choppy and flickery); I thought it was better to have it work well or not at all. (In unsupported browsers there is still an overlay color, so I think it gets the point across.)